### PR TITLE
Update Update-MpSignature.md

### PR DESCRIPTION
**Pull request by AirTudor**
Submitted via Learn Editor.

### CODE REVIEW

**Comment by AirTudor** [comment]
#sign-off

File: docset/winserver2016-ps/defender/Update-MpSignature.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ PS C:\> Update-MpSignature
 ```
 
 This command updates the antimalware definitions.
-By default, the cmdlet uses the default update source.
+By default, the cmdlet uses the sources configured through [SignatureFallbackOrder](Set-MpPreference.md#-signaturefallbackorder). If no signature fallback order is configured, then the cmdlet uses the default update source.
```
Comment (JohanFreelancer9) [inline docset/winserver2016-ps/defender/Update-MpSignature.md:34]:
```suggestion
By default, the cmdlet uses the sources configured through [SignatureFallbackOrder](Set-MpPreference.md#-signaturefallbackorder). If no signature fallback order is configured, the cmdlet uses the default update source.
```

File: docset/winserver2016-ps/defender/Update-MpSignature.md [hunk 2]
```diff
@@ -101,7 +101,7 @@ Accept wildcard characters: False
 ### -UpdateSource
 Specifies an update source.
 This cmdlet downloads updated definitions from the server you specify.
-If you do not specify this parameter, the cmdlet uses the Microsoft Update Server, then Microsoft Malware Protection Center (MMPC), as an update source.
+If you do not specify this parameter, the cmdlet uses the sources configured through [SignatureFallbackOrder](Set-MpPreference.md#-signaturefallbackorder). If no signature fallback order is configured, then the cmdlet uses Microsoft Update Server, then Microsoft Malware Protection Center (MMPC), as an update source.
```
Comment (JohanFreelancer9) [inline docset/winserver2016-ps/defender/Update-MpSignature.md:104]:
```suggestion
If you do not specify this parameter, the cmdlet uses the sources configured through [SignatureFallbackOrder](Set-MpPreference.md#-signaturefallbackorder). If no signature fallback order is configured, the cmdlet uses Microsoft Update Server, then Microsoft Malware Protection Center (MMPC) as an update source.
```

File: docset/winserver2019-ps/defender/Update-MpSignature.md [hunk 3]
```diff
@@ -31,7 +31,7 @@ PS C:\> Update-MpSignature
 ```
 
 This command updates the antimalware definitions.
-By default, the cmdlet uses the default update source.
+By default, the cmdlet uses the sources configured through [SignatureFallbackOrder](Set-MpPreference.md#-signaturefallbackorder). If no signature fallback order is configured, then the cmdlet uses the default update source.
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/defender/Update-MpSignature.md:34]:
```suggestion
By default, the cmdlet uses the sources configured through [SignatureFallbackOrder](Set-MpPreference.md#-signaturefallbackorder). If no signature fallback order is configured, the cmdlet uses the default update source.
```

File: docset/winserver2022-ps/defender/Update-MpSignature.md [hunk 4]
```diff
@@ -31,7 +31,7 @@ PS C:\> Update-MpSignature
 ```
 
 This command updates the antimalware definitions.
-By default, the cmdlet uses the default update source.
+By default, the cmdlet uses the sources configured through [SignatureFallbackOrder](Set-MpPreference.md#-signaturefallbackorder). If no signature fallback order is configured, then the cmdlet uses the default update source.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/defender/Update-MpSignature.md:34]:
```suggestion
By default, the cmdlet uses the sources configured through [SignatureFallbackOrder](Set-MpPreference.md#-signaturefallbackorder). If no signature fallback order is configured, the cmdlet uses the default update source.
```

File: docset/winserver2022-ps/defender/Update-MpSignature.md [hunk 5]
```diff
@@ -101,7 +101,7 @@ Accept wildcard characters: False
 ### -UpdateSource
 Specifies an update source.
 This cmdlet downloads updated definitions from the server you specify.
-If you do not specify this parameter, the cmdlet uses the Microsoft Update Server, then Microsoft Malware Protection Center (MMPC), as an update source.
+If you do not specify this parameter, the cmdlet uses the sources configured through [SignatureFallbackOrder](Set-MpPreference.md#-signaturefallbackorder). If no signature fallback order is configured, then the cmdlet uses Microsoft Update Server, then Microsoft Malware Protection Center (MMPC), as an update source.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/defender/Update-MpSignature.md:104]:
```suggestion
If you do not specify this parameter, the cmdlet uses the sources configured through [SignatureFallbackOrder](Set-MpPreference.md#-signaturefallbackorder). If no signature fallback order is configured, the cmdlet uses Microsoft Update Server, then Microsoft Malware Protection Center (MMPC) as an update source.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @AirTudor, kindly commit the suggested changes. Many thanks.

**Comment by garycentric** [comment]
@AirTudor , @denisebmsft 

@traya1 has opened a service ticket with SiteHelp for the blocking issues in this pull request.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by traya1** [comment]
Hi Team,

We have a solution for the link-out-of-scope errors but it introduces a new issue with the bookmark references. I have communicated the new issue with the support team and will helpfully hear back soon regarding a fix for the issue.

**Comment by traya1** [comment]
@AirTudor , @denisebmsft

The issue has been resolved. I have removed the PubOps labels so y'all can review the changes, If all looks good, please re-apply Sign off and we will review/merge.

Let me know if you have any more questions or concerns.

**Comment by traya1** [comment]
Hi @xmdanni - It seems that the update works. I'll get the links in this PR fixed and will work with my team to update any other affected links in the repo.

File: docset/winserver2016-ps/defender/Update-MpSignature.md [hunk 6]
```diff
@@ -31,7 +31,7 @@ PS C:\> Update-MpSignature
 ```
 
 This command updates the antimalware definitions.
-By default, the cmdlet uses the default update source.
+By default, the cmdlet uses the sources configured through [SignatureFallbackOrder](/powershell/module/defender/set-mppreference?view=windowsserver2016-ps#-signaturefallbackorder). If no signature fallback order is configured, the cmdlet uses the default update source.
```
[the remark this answers is not in the document]
Comment (traya1) [reply docset/winserver2016-ps/defender/Update-MpSignature.md]: update has been made

**Review by get-itips — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@tiburd 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**